lib/demangler.js

```diff
@@ -148,12 +178,21 @@ class Demangler extends AsmRegex {
         for (let i = 0; i < this.result.asm.length; ++i) {
             let line = this.result.asm[i].text;
             for (let j = 0; j < translations.length; ++j) {
-                line = line.replace(translations[j][0], translations[j][1]);
-                line = line.replace(translations[j][0], translations[j][1]);
+                const oldValue = translations[j][0];
+                const newValue = translations[j][1];
+
+                line = line.replace(oldValue, newValue);
```

Comment:
I realise this is the same behaviour as before but I will take a note to investigate why this replacement had to be done twice...thanks

Reply:
I think we need to call the `replace` function twice because it is possible that a source line contains the same label twice. For example:
https://github.com/mattgodbolt/compiler-explorer/blob/8926e87522a9d8ce1708940e2c63b540e8566d4d/test/cases/arm-jump-table.asm#L6

Reply:
Agreed, but "surely" there's a better way to do this...what if the label appears three times? :) with regex searches we can use `/g` modifier...maybe there's not one for regular string replace...